pythonbits/bb.py

```diff
@@ -73,7 +79,10 @@ def subcategorise(self):
     @staticmethod
     def submit(payload):
         t = Tracker()
-        return t.upload(**payload)
+        try:
+            return t.upload(**payload)
+        except TrackerException as e:
+            log.error('Upload failed: %s' % (e,))
```

Comment:
Why prevent the error from being raised?

Reply:
Because throwing tracebacks at the user isn't very nice. But I guess errors should be caught somewhere in __main__(). I'll look into it.

Reply:
I removed that exception catcher here. `__main__` is actually catching exceptions, it's just printing it with the traceback. Maybe I'll try to tell logbook to only print the traceback when debugging is enabled or something.